chromadb/auth/token_authn/__init__.py

```diff
@@ -232,4 +234,4 @@ def authenticate_or_raise(self, headers: Headers) -> UserIdentity:
         time.sleep(
             random.uniform(0.001, 0.005)
         )  # add some jitter to avoid timing attacks
-        raise HTTPException(status_code=403, detail="Forbidden")
+        raise self.HTTPException(status_code=403, detail="Forbidden")
```

Comment:
I think we could probably just import it here instead of attaching it to the class?